generator/templates/scss/scss/_variables.scss

```diff
@@ -0,0 +1,152 @@
+// Included below are all the defined variables from Bulma

```

Comment:
It is not a good idea to bring some buefy code to this project because we are duplicating code and we will have to keep it updated as well.